jira/client.py

```diff
@@ -1785,8 +1777,7 @@ def assign_issue(self, issue: Union[int, str], assignee: Optional[str]) -> bool:
         url = self._get_latest_url(f"issue/{issue}/assignee")
         user_id = self._get_user_id(assignee)
         payload = {"accountId": user_id} if self._is_cloud else {"name": user_id}
-        r = self._session.put(url, data=json.dumps(payload))
-        raise_on_error(r)
+        self._session.put(url, data=json.dumps(payload))
```

Comment:
no more raise?
or is that a given?
that could result maybe in different behavior in the parts where this raise wasnt specifically called...

Reply:
not required even in the previous implementation as we already called it internally.